README.md

```diff
@@ -44,26 +48,48 @@ You can also download from [v15-dev releases](https://github.com/devendrn/newb-x
 - Python packages:
   - [lazurite](https://veka0.github.io/lazurite/#installation) (must be v0.2.1. newer or older version may not be supported)
   - [rich](https://rich.readthedocs.io/en/stable/introduction.html#installation) (must be v13.x.x)
+> [!NOTE]
+> Newer versions may not work as expected. In this case install from requirements.txt
+
+<br>
 
 ### Get source code
 ```
 git clone https://github.com/devendrn/newb-x-mcbe/
 cd newb-x-mcbe
 ```
 
+<br>
+
+### Install dependencies from requirements.txt
+*Skip if you already have installed those versions.*
+```
+python -m pip install -r requirements.txt
+```
+
+> [!TIP]
+> If you have multiple Python versions on Linux like python3, python3.11, python3.12 etc and want to use specific version for compiling `let's say python3.11`. Open up build.sh file. Change **python3** to **python3.11** and that version will be used for all the scripts.
+
+<br>
+
 ### Setup build environment
 > [!NOTE]
-> On Windows, you should use `.\build.bat` instead of `./build.sh` for all following commands. 
+> On Windows, run `build.bat` instead of `./build.sh` for all following commands. 
 ```
 ./build.sh setup
 ```
 This will download shaderc binary and material data required to build shader.
 
+<br>
+
 ### Compile specific shader materials
 ```
 ./build.sh mats
-```  
-Compiled material.bin files will be inside `build/<platform>/`  
+```
+
+Compiled material.bin files will be inside `build/<platform>/`
+
+

```

Comment:
Is there a need for newlines here?

Reply:
I did it for looks. But removed extra lines in new pull